### PR TITLE
Fix bug: Show all items when "All" tab is active

### PR DESCRIPTION
**Pull request by Totemika**
Small change to show all items when the "All" tab is active.

This way the behavior is the same as in the AngularJS version of the todos app.

### CODE REVIEW

File: examples/typescript-angular/js/controllers/TodoCtrl.ts [hunk 1]
```diff
@@ -50,9 +50,9 @@ module todos {
 		}
 
 		onPath(path: string) {
-			this.$scope.statusFilter = (path === '/active') ?
-				{ completed: false } : (path === '/completed') ?
-				{ completed: true } : null;
+            this.$scope.statusFilter = (path === '/active') ?
```
Comment (passy) [inline examples/typescript-angular/js/controllers/TodoCtrl.ts]: This looks like spaces. :)

Comment (passy) [inline examples/typescript-angular/js/controllers/TodoCtrl.ts]: Oh yeah, already fixed! :+1:

**Comment by dmethvin** [comment]
Should there be a test for this?

**Comment by samccone** [comment]
yeah, I guess we do not have one :P @dmethvin mind opening another issue to track this?

File: examples/typescript-angular/js/interfaces/ITodoScope.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ module todos {
 		doneCount: number;
 		allChecked: boolean;
 		reverted: boolean;
-		statusFilter: { completed: boolean; };
+		statusFilter: any;
```
Comment (passy) [inline examples/typescript-angular/js/interfaces/ITodoScope.ts]: Is there a way to specify this more narrowly? Not good with typescript, but could we mark `completed` as optional key?

Comment (dmethvin) [inline examples/typescript-angular/js/interfaces/ITodoScope.ts]: Maybe `{ completed?: boolean; }` instead?

**Comment by samccone** [comment]
ps @Totemika Thanks so much for this PR :)

File: examples/typescript-angular/js/interfaces/ITodoScope.ts (hunk 2 above)
Comment (Totemika) [inline examples/typescript-angular/js/interfaces/ITodoScope.ts]: Yeah, I like that better too.

**Comment by Totemika** [comment]
:open_mouth:  thank you guys for the whole project and helping me learn :smile:

**Comment by sindresorhus** [comment]
Nice work on the PR @Totemika. Looks good to me when the current feedback is resolved :)

**Comment by passy** [comment]
Fabulous work, @Totemika! :100: Thanks so much!

**Pull request closed without merging**